zkevm-circuits/src/table.rs

```diff
@@ -1310,3 +1310,128 @@ impl<F: Field> LookupTable<F> for ExpTable {
         ]
     }
 }
+
+/// Keccak Table, used to verify keccak hashing from RLC'ed input.
+#[derive(Clone, Debug)]
+pub struct KeccakTable2 {
+    /// True when the row is enabled
+    pub is_enabled: Column<Advice>,
+    /// Byte array input as `RLC(reversed(input))`
+    pub input_rlc: Column<Advice>, // RLC of input bytes
+    /// Byte array input length
+    pub input_len: Column<Advice>,
+    /// hash high 16 bytes
+    pub output_hi: Column<Advice>,
+    /// hash low 16 bytes
+    pub output_lo: Column<Advice>,
```

Comment:
Nice, seems we're going for future proof here. :)
But may also make it a bit more challenging in the future to add additional circuits, but I guess that's okay. We're not trying to create a sound circuit implementation anyway so we have some flexibility.